bundle/edu.gemini.ags/src/test/scala/edu/gemini/ags/impl/Helpers.scala

```diff
@@ -0,0 +1,32 @@
+package edu.gemini.ags.impl
+
+import edu.gemini.ags.conf.ProbeLimitsTable
+import edu.gemini.skycalc.{DDMMSS, HHMMSS}
+import edu.gemini.spModel.core.{Declination, RightAscension, Angle, Coordinates, MagnitudeBand, Magnitude, SiderealTarget}
+import edu.gemini.spModel.target.SPTarget
+
+/**
+ *
+ */
+trait Helpers {
+  val mt       = ProbeLimitsTable.loadOrThrow()
+  val zeroBase = basePosition("00:00:00.000 00:00:00.00")
```

Comment:
Almost, but we need an `SPTarget` here which is what the `basePosition` helper provides.